index.html

```diff
@@ -2271,6 +2262,15 @@ <h2 id="OfflineAudioContext">
               for the occasion.
             </p>
             <ol>
+              <li>Create a new <code>AudioBuffer</code>, with a number of
```

Comment:
Initialize a slot here, and the store the `AudioBuffer` in the slot.

We need to add language to the `AudioBuffer` ctor to allow throwing OOM errors and returning undefined or null or something.